projects/fsastorefrontlib/src/cms-components/checkout/components/configure-product/navigation/product-configuration-navigation.component.spec.ts

```diff
@@ -91,11 +108,15 @@ describe('ProductConfigurationNavigationComponent', () => {
   let pricingService: PricingService;
   let currentProductService: CurrentProductService;
   let cartService: FSCartService;
+  let routingService: RoutingService;
 
   beforeEach(
     waitForAsync(() => {
       TestBed.configureTestingModule({
-        declarations: [ProductConfigurationNavigationComponent],
+        declarations: [
+          ProductConfigurationNavigationComponent,
+          MockTranslatePipe,
```

Comment:
There is no need to mock translate pipe, we use it in almost all components. 
Just import I18nTestingModule.

Reply:
Actually there is no need even for `I18nTestingModule `, when I got error for pipe it wasn't related to this pipe. I'll just remove `MockTranslatePipe`.